DQMOffline/Configuration/python/DQMOffline_Certification_cff.py

```diff
@@ -45,5 +45,9 @@
     sipixelCertification # segfaults when included
 ]))
 
+phase1Pixel.toReplaceWith(DQMCertTrackerPixel, DQMCertTrackerPixel.copyAndExclude([ # FIXME
+    sipixelCertification # segfaults when included
```

Comment:
I believe the change is due to these lines I had to introduce to avoid the crash with the new configuration, since the lines above were removing Pixel Certification in Phase1 wf, but probaby in a not effective way